### PR TITLE
New package: TranslucentTB.TranslucentTB version 2023.1

### PR DESCRIPTION
**Pull request by coolplaylinbot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120647)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @coolplaylinbot and @CoolPlayLin,

This package appears to have a dependency. I'm not sure, but I believe it might be `VCRedist`:
image

**Comment by stephengillie** [comment]
This package needs a `MinimumOSVersion` above 22000. 
image

**Comment by CoolPlayLin** [comment]
@stephengillie It fails

File: manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TranslucentTB.TranslucentTB
+PackageVersion: '2023.1'
+InstallerType: zip
+NestedInstallerType: portable
+MinimumOSVersion: 22000.0.0.0
```
Comment (stephengillie) [inline manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml:8]:
```suggestion
MinimumOSVersion: 10.0.22621.0
```

**Comment by CoolPlayLin** [comment]
The validation pipeline has recently failed a lot😒

**Comment by CoolPlayLin** [comment]
@stephengillie Oh, it fails again......

**Comment by CoolPlayLin** [comment]
@stephengillie the same error appears

**Review by CoolPlayLin — APPROVED** [review]
It installs normal in my machine

image

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by CoolPlayLin** [comment]
## For moderators
This account is bot account and belongs to @CoolPlayLin. Any issue should call him up instead of assign to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

Have a good day!

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

**Comment by CoolPlayLin** [comment]
This software is used to make your sidebar have special effect like transparent. I don't know why it throws error.

CC @sylveon

**Comment by sylveon** [comment]
I don't recommend the portable version unless you really need it, just get it from the store. WinGet can already do that.

I don't know why it's failing on your machine however, it's the first time I've seen this error code.

**Comment by CoolPlayLin** [comment]
@stephengillie Try to install it in a physical machine instead of virtual machines.

File: manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        PortableCommandAlias: TranslucentTB
+  - Architecture: arm64
+    InstallerType: zip
+    NestedInstallerType: portable
+    InstallerUrl: https://github.com/TranslucentTB/TranslucentTB/releases/download/2023.1/TranslucentTB-portable-arm64.zip
+    InstallerSha256: BA5213D9FF7B60489BE80FDB58A357F6D83DD02F1FF7BCF25C71A3CC3E10FC0F
+    MinimumOSVersion: 10.0.22621.0
+    Dependencies:
+      PackageDependencies:
+        - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (sylveon) [inline manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml:29]: This is wrong.

File: manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    NestedInstallerFiles:
+      - RelativeFilePath: TranslucentTB.exe
+        PortableCommandAlias: TranslucentTB
+  - Architecture: x86
```
Comment (sylveon) [inline manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml:33]: The MSIX has no support for either x86 or ARM, it's only x64 and ARM64

File: manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml (hunk 2 above)
Comment (CoolPlayLin) [reply manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml:29]: According to the Microsoft's docs
image

Comment (sylveon) [reply manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml:29]: Just use the ARM64 redist.

**Pull request closed without merging**